WordPress/Classes/ApplicationToken/ApplicationTokenListView.swift

```diff
@@ -0,0 +1,149 @@
+import SwiftUI
+import DesignSystem
+import WordPressUI
+import WordPressAPI
+
+struct ApplicationTokenListView: View {
+
+    @ObservedObject
+    private var viewModel: ApplicationTokenListViewModel
+
+    fileprivate init(tokens: [ApplicationTokenItem]) {
+        let dataProvider = StaticTokenProvider(tokens: .success(tokens))
+        self.init(viewModel: ApplicationTokenListViewModel(dataProvider: dataProvider))
```

Comment:
`@ObservedObject` doesn't make any guarantees about the lifetime of the object. It's expected that the object lifetime is managed elsewhere, and it's injected in the view. If it's not injected, the view has to use `@StateObject`.

`@StateObject` has its own caveats. Please refer to https://developer.apple.com/documentation/swiftui/stateobject#Initialize-state-objects-using-external-data.

Reply:
This initializer and the `init(error:)` one blow are private functions for SwiftUI preview. The `init(viewModel:)` is meant for external use, which is why `viewModel` is declared as `@ObservedObject`.